### PR TITLE
Removes Christmas. 

### PR DESCRIPTION
**Pull request by Probe1**
## Ready for Review/Merge.

download 2

### CODE REVIEW

File: maps/tgstation.dmm [hunk 1]
```diff
@@ -967,7 +967,7 @@
 "asE" = (/obj/machinery/atmospherics/unary/vent_pump{dir = 8; level = 2; on = 1},/turf/simulated/floor,/area/security/brig)
 "asF" = (/obj/structure/closet/secure_closet/brig{id_tag = "Cell 2"; name = "Cell 2 Locker"},/obj/machinery/camera{c_tag = "Brig Cell 2"},/turf/simulated/floor{icon_state = "red"; dir = 4},/area/security/brig)
 "asG" = (/obj/structure/cable{d1 = 1; d2 = 2; icon_state = "1-2"},/turf/simulated/floor{icon_state = "red"; dir = 8},/area/security/lobby)
-"asH" = (/obj/structure/cable{d1 = 4; d2 = 8; icon_state = "4-8"},/obj/structure/window,/turf/simulated/floor,/area/security/lobby)
```
Comment (ghost) [inline maps/tgstation.dmm:970]: Cables removed, this alright?

Comment (Probe1) [reply maps/tgstation.dmm:970]: Funny how doing this all en masse resulted in no mistakes but the windows I deleted by hand I somehow fucked up on!

**Comment by 9600bauds** [comment]
Now let's all agree to never talk about 2016 again.

File: maps/tgstation.dmm (hunk 1 above)
Comment (Probe1) [reply maps/tgstation.dmm:970]: https://a.pomf.cat/exztlp.png  I'll be damned if I can find this mysterious wire.

Comment (Probe1) [reply maps/tgstation.dmm:970]: Best guess is I copied and pasted it for some reason when mapping it for the christmas update.  Everythings five by five in game and in dreammaker.

**Comment by Exxion** [comment]
Done manually, right?
Also, Easter eggs?

**Comment by Probe1** [comment]
Manually in the sense that I replaced all the christmas shit with nothing using Dreammakers built in search find replace function, then compiled and scoured the maps for any mistakes or missed items.

I nearly forgot the gingerbread man.  I feel very confident since I kept a checklist of what we did https://github.com/Probe1/vgstation13/issues/9

**Comment by PJB3005** [comment]
👊

**Pull request merged**